docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -210,81 +204,64 @@ ifeval::["{build}" == "downstream"]
 3b14cc7bf4dd  registry.redhat.io/ceph/rhceph@sha256:320c364dcc8fc8120e2a42f54eb39ecdba12401a2546763b7bef15b02ce93bc4  -n mgr.oc0-contro...  35 minutes ago  Up 35 minutes ago              ceph-f6ec3ebe-26f7-56c8-985d-eb974e8e08e3-mgr-oc0-controller-1-mtxohd
 endif::[]
 ----
-
-To cleanup the source node before moving to the next phase, cleanup the existing
-containers and remove the cephadm related data from the node.
+ifeval::["{build}" == "downstream"]
+To clean up the existing containers and remove the `cephadm` data from the source node, contact Red Hat Support.
+endif::[]
+====
 // fpantano: there's an automated procedure run through cephadm but it's too
 // risky. If the user doesn't perform it properly the cluster can be affected.
```

Comment:
I added an ifeval just above this hidden comment that I think addresses this. Please review and let me know if that is sufficient, and then I can remove the comment.